spec/models/family_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+describe Family do
+  describe "validations" do
+    context "is valid only if last_name and capability are present" do
+
+      it { is_expected.to validate_presence_of(:last_name) }
+      it { is_expected.to validate_presence_of(:capability) }
```

Comment:
A question: should we validate that capability is an integer?

Reply:
I suppose yes but with the simple_form you have a drop box where you can only pick a number. Do you think we still need to test it?